### PR TITLE
Added null_resource

### PR DESCRIPTION
**Pull request by jackluo923**
Added [null_resource resource](https://registry.terraform.io/providers/hashicorp/null/latest/docs/resources/resource) updated documents according to the contributing guidelines.

### CODE REVIEW

File: terrascript/resource/__init__.py [hunk 1]
```diff
@@ -46,6 +46,7 @@
 from .newrelic import *
 from .nomad import *
 from .ns1 import *
+from .null import *
```
Comment (ilons) [inline terrascript/resource/__init__.py:49]: Note to self: we REALLY should stop doing star imports :/

File: terrascript/resource/null.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# terrascript/resource/null.py
+import terrascript
```
Comment (ilons) [inline terrascript/resource/null.py:2]: Please run `make black` and commit changes (should be line breaks here i suspect)

**Review by ilons — CHANGES_REQUESTED** [review]

File: terrascript/resource/__init__.py (hunk 1 above)
Comment (mjuenema) [reply terrascript/resource/__init__.py:49]: I agree. I can remove them at least from ``resource/__init__.py``, ``provider/__init__.py`` and ``data/__init__.py`` for release 0.10.

File: terrascript/resource/null.py (hunk 2 above)
Comment (jackluo923) [reply terrascript/resource/null.py:2]: Done

**Comment by ilons** [comment]
I'll help get this merged when at a computer again (late this week).

**Review by ilons — APPROVED** [review]

**Pull request merged**